bonnie-rest/src/main/java/com/cgi/bonnie/bonnierest/OrderController.java

```diff
@@ -53,6 +53,7 @@ public ResponseEntity<List<Order>> getAllOrders() {
     public ResponseEntity<String> createOrder(@RequestBody OrderRequest request) {
         try {
             Status status = Status.valueOf(request.getStatus().toUpperCase());
+//fixme order without shopOrderId, placementDate and metadata
```

Comment:
@KenyeresKornel also can be deleted?

Reply:
This endpoint is going to be used by the Admin if there will be an opportunity for him to place orders in the name of the shop (also pretty good for testing, but I don't mind if you delete it)

Reply:
So it is necessary.
Then OrderRequest also has to be fixed with tree missing fields